config/linux_gpu.cmake

```diff
@@ -41,6 +41,7 @@
 #---------------------------------------------
 set(USE_CUDA ON CACHE BOOL "Build with CUDA support")
 set(USE_CUDNN ON CACHE BOOL "Build with cudnn support, if found")
+set(USE_CUTENSOR OFF CACHE BOOL "Build with cutensor support, if found")
```

Comment:
Let's turn this `ON`?

Reply:
Revised.

Reply:
It seems you haven't pushed the change in linux_gpu.cmake?

Reply:
Why revert this commit?